Backend/DatabaseRequester.py

```diff
@@ -0,0 +1,58 @@
+import psycopg2
+from configparser import ConfigParser
+
+class DatabaseRequester:
+
+    conn = None
+    connected = False
+    host = None
+    port = None
+    database = None
+    user = None
+    password = None
```

Comment:
I think this is declaring static variables (quirk of python). the code inside the constructor (init) is separately declaring instance variables with the same name. Is that intended?